src/main/scala/uk/gov/hmrc/hmrcfrontend/config/BackLinkConfig.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.hmrcfrontend.config
+
+import play.api.Configuration
+
+import javax.inject.Inject
+
+class BackLinkConfig @Inject() (configuration: Configuration) {
+  def enableBrowserBack: Boolean =
+    configuration.getOptional[Boolean]("hmrc-back-link.enable-browser-back-if-no-url").getOrElse(true)
```

Comment:
`getOrElse(true)` implies it's on by default, to ease adoption, but may break services already rolling their own solutions unless they add config to switch off (or migrate to our solution)

Reply:
removed this now